### PR TITLE
June ISM Compliance updates

### PR DESCRIPTION
**Pull request by cwesten-CyberCX**
Compliance updates to the security documentation to include both June 2021 ISM updates and changes within the Blueprints design and configuration. 
Updates include -

- ISM June 2021 Updates
- iOS 14 ACSC Hardening Guidance update, including Per-app VPN and managing how applications are installed to iOS devices
- Local administrator account updates
- Blocking of Inbound and Outbound traffic from Windows 10 endpoints using Windows Defender Firewall

### CODE REVIEW

File: blueprint/security/hybrid-security-risk-management-plan.md [hunk 1]
```diff
@@ -1290,13 +1290,13 @@ Administration of the system is undertaken by authorised privileged users by con
...
-### R17 Mobile device compromised
+### R17 Mobile phone compromised due to loss or theft
 
 #### Risk overview
 
-An Apple iOS device used to access Sensitive and or security classified data is compromised as a result of the [ACSC’s Security Configuration Guide - Apple iOS 12 Devices](https://www.cyber.gov.au/publications/security-configuration-guide-apple-ios-12-devices) (September 2019)  not being fully implemented due to the usability impacts. 
+An Apple iOS device used to access Sensitive and or security classified information is compromised as a result of the device being lost or stolen.
 
-Note, the HybridSystem does not include the use of devices using the Android operating system.
+Note: the CloudSystem does not include the use of devices using the Android operating system.
```
Comment (zedzedzed) [inline blueprint/security/hybrid-security-risk-management-plan.md:1299]: `CloudSystem` should be `HybridSystem`. There are 2 more references in this document.

File: blueprint/security/system-security-plan.md [hunk 2]
```diff
@@ -502,9 +502,9 @@ This section is applicable as the CloudSystem contains Microsoft Teams which pro
...
+The CloudSystem inherits the security controls Microsoft have implemented for Microsoft Teams as assessed in the Microsoft 365 IRAP report.
 
-The Annex specifies the controls associated with the self-contained use of Microsoft Teams up to the level of PROTECTED. As per the HybridSystem solution design, agencies have the ability to connect Teams to the Telstra Calling for Office 365 service to allow calling between Teams and traditional telephones. Agencies wishing to use Telstra Calling for Office 365 or another similar services should undertake a security assessment to ensure that the product addresses their security requirements.
+The Annex specifies the controls associated with the self-contained use of Microsoft Teams up to the level of PROTECTED. As per the HybridSystem solution design, agencies have the ability to connect Teams to the Telstra Calling for v 365 service to allow calling between Teams and traditional telephones. Agencies wishing to use Telstra Calling for Microsoft 365 or another similar services should undertake a security assessment to ensure that the product addresses their security requirements.
```
Comment (zedzedzed) [inline blueprint/security/system-security-plan.md:507]: Telstra Calling for `v` 365 service

v = Microsoft ?

File: blueprint/security/hybrid-security-risk-management-plan.md (hunk 1 above)
Comment (cwesten-CyberCX) [reply blueprint/security/hybrid-security-risk-management-plan.md]: Updated

File: blueprint/security/system-security-plan.md (hunk 2 above)
Comment (cwesten-CyberCX) [reply blueprint/security/system-security-plan.md]: Updated

**Comment by zedzedzed** [comment]
Looks good. To merge, relies on #66, #68 and #71

**Pull request closed without merging**